dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -193,3 +194,24 @@ def gini_impurity(self):
             gini_sum += (self._categories[i]/self.sample_size) * \
                          (1 - (self._categories[i]/self.sample_size))
         return gini_sum
+
+    @property
+    def unalikeability(self):
+        """
+        Property for Unlikeability. Unikeability checks for
+        "how often observations differ from one another"
+
+        U = Σ(i=1,n)Σ(j=1,n): (Cij)/n**2
```

Comment:
let us add the reference to the resource:
`Perry, M. and Kader, G. Variation as Unalikeability. Teaching Statistics, Vol. 27, No. 2 (2005), pp. 58-60.`

we need to use the version: U = Σ(i=1,n)Σ(j=1,n): (Cij)/(n**2 - n)
otherwise this and gini-impurity are the same